purchase_last_price_info/hooks.py

```diff
@@ -1,15 +0,0 @@
-# Copyright 2019 ForgeFlow S.L.
-#   (http://www.forgeflow.com)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
-from odoo import SUPERUSER_ID, api
-
-
-def set_last_price_info(cr, registry):
```

Comment:
Can you clarify why this is not needed anymore?

Reply:
It's not necessary because now `last_purchase_line_id` field is auto-set wit all info.

Reply:
Ok, then maybe put that information in the commit message body and in the PR description.